MAUI/PDF-Viewer/Toolbar.md

```diff
@@ -47,7 +47,25 @@ To customize the toolbar, you can programmatically add, remove, and hide the ite
 
 ### Hide specific toolbar
 
-By using the [GetByName](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.ToolbarCollection.html#Syncfusion_Maui_PdfViewer_ToolbarCollection_GetByName_System_String) method with its specified toolbar names, you can access the properties of those toolbars. After accessing the toolbar, you can modify its visibility by setting the `IsVisible` property. Here, we retrieve the toolbar "BottomToolbar" using the GetByName method and hide it by setting its visibility to false.
+Sometimes, you might need to hide specific toolbars instead of all. This can be useful if you want to simplify the user interface by removing unnecessary tools or creating a more focused environment for certain tasks. The Toolbars collection property in the PDF Viewer allows you to hide a specific toolbar by using its index or name.
```

Comment:
What are the changes made for this query?

<img width="650" alt="image" src="https://github.com/user-attachments/assets/f1cd324c-048e-4c83-b44b-22ca682fb71f">

Reply:
I have organised

Remove Items from the Toolbar
Remove Item by Index
Remove Item by Name
Hide specific toolbar
Hide Toolbars by Name 
Hide Toolbars by Index